### PR TITLE
Relativistic Vampyr for bielecgtronic atoms with coulomb and guant us…

### PR DESCRIPTION
**Pull request by Christian48596**
…ing in input argumnets

### CODE REVIEW

File: orbital4c/NuclearPotential.py [hunk 1]
```diff
@@ -2,15 +2,16 @@
 import copy as cp
 from scipy.special import erf
 
-def CoulombPotential(position, center, charge):
+
+def PoCh(position, center , charge):
```
Comment (ilfreddy) [inline orbital4c/NuclearPotential.py:6]: Could you please use longer names? "PoCh" --> "CoulombPotential"
As I told you earlier, your future self will thank you!

File: orbital4c/NuclearPotential.py [hunk 2]
```diff
@@ -26,13 +27,6 @@ def uHFYGB(r):
...
     return u
 
-def PoCh(position, center , charge):
-    d2 = ((position[0] - center[0])**2 +
-          (position[1] - center[1])**2 +
-          (position[2] - center[2])**2)
-    distance = np.sqrt(d2)
-    return charge / distance
-
 def HomChSph(position, center, charge, atom):
```
Comment (ilfreddy) [inline orbital4c/NuclearPotential.py:30]: Same here: 
e.g.  HomChSph --> PotentialHomogeneousSphere
and other places in the file with too short names :-)

**Review by ilfreddy — COMMENTED** [review]
Please fix the name stuff :-)

File: orbital4c/NuclearPotential.py (hunk 2 above)
Comment (Christian48596) [reply orbital4c/NuclearPotential.py]: Fixed and pushed

**Pull request merged**